src/utils/validators.ts

```diff
@@ -11,6 +11,8 @@ const isEnsTest = (value: string) => value.endsWith('.eth') && value.length > 6;
 const isAnyTest = (value: string) => value === 'any';
 
 const isPositiveIntegerStringTest = (value: string) => /^\d+$/.test(value);
+const isBetween10And1000Test = (value: string) =>
+  /^(?!0*$)([1-9]\d{0,2}|1000)$/.test(value);
```

Comment:
Regex quite hard to read, and it looks like '1' is valid whether it should not?

<img width="615" alt="Capture d’écran 2023-11-21 à 10 26 17" src="https://github.com/iExecBlockchainComputing/dataprotector-sdk/assets/4280765/ddd76a3e-a2d7-4cd9-9447-cdbb596dcad8">

I would suggest:

```
export const numberBetweenSchema = (min: number, max: number) =>
  number()
    .integer()
    .min(min)
    .max(max)
    .typeError(`$\{path} must be a number between ${min} and ${max}`);
```